docs/extensions/info-xml.md

```diff
@@ -291,7 +291,7 @@ Elements acceptable within `<requires>`
 ### `<tags>` {:#tags}
 
 * Containing element: [`<extension>`](#extension)
-* Description: Freeform tags -but not that there are guideline as to what to assign
+* Description: Freeform tags. By convention, tag-names should be "snake-case" (all lower-case alphanumeric with dashes for separate words). If namespaces or prefixes are required, tag-names may use a colon (`:`) delimiter. Tag-names should be documented in this file. (*If you need to use a new/unknown tag, then please submit an update PR to this documentation with a description/purpose.*)
```

Comment:
Suggest changing "Tag-names should be documented in this _file_" to "Tag-names should be documented in this _page_". On first reading, I thought "file" was referring to the info.xml _file_, which is after all the topic of this _page_.

Reply:
Ah +1. That's better wording.